feeadjuster/feeadjuster.py

```diff
@@ -13,45 +13,51 @@
 plugin.update_threshold = 0.05
 
 
-def get_ratio(our_percentage):
+def get_ratio(plugin: Plugin, our_percentage):
     """
     Basic algorithm: the farther we are from the optimal case, the more we
     bump/lower.
     """
-    return 50**(0.5 - our_percentage)
+    return plugin.ratio_base**(0.5 - our_percentage)
```

Comment:
This function was basically a handcrafted hack. I [expected](https://github.com/lightningd/plugins/pull/147#issuecomment-686434101) to (didn't have time yet though..) add more variants of the `get_ratio` function with more variables taken into account.
The user experience would be something like this in the config:
```
adjustment_method="basic" # The default
# Or
adjustment_method="htlc-weighted"
# Or
adjustment_method="funder-weighted"
# Or
adjustment_method="soft"
# Etc...
```
I think that tweaking the handcrafted hack is not the best way to go.

Could we instead have a `get_ratio_soft` and `get_ratio_hard` which would respectively have higher hard-coded constants instead of exposing the internals to users ?